wp-content/mu-plugins/pantheon/pantheon-updates.php

```diff
@@ -64,7 +111,8 @@ function _pantheon_upstream_update_notice() {
 // Register Pantheon specific WordPress update admin notice
 add_action( 'admin_init', '_pantheon_register_upstream_update_notice' );
 function _pantheon_register_upstream_update_notice(){
-    // but only if we are on Pantheon and there is a WordPress update available
+	// but only if we are on the Pantheon dev environment
```

Comment:
Only if we are on Pantheon

Reply:
Ah, yes, the ol' "changed the code but not the comments".